### PR TITLE
Changes stunshells to fire like buckshot

### PR DESCRIPTION
**Pull request by jakmak5**
As it stands now stunshells are just tazer rounds for shotguns. They do the same stun and fire just like a tazer while costing 3000 metal to make in the protolathe (the stun revolver costs 6000 metal to make) This ofcourse makes them the same as beanbag. This PR causes them to fire 4 stun projectiles just like buckshot does. Also reduces their price in the protolathe from 3000 to 1500 per shell and for the price of some cardboard you can make a box of 15 from the protolathe for 6 sheets of iron. Note the stun does not stack if you are hit by multiple projectiles from the same shell. 

webm of it in action

https://puu.sh/tSx1Y/0ccdc6de20.webm

note on hit for some reason the stun shells do 5 damage on hit. If requested i can remove that

### CODE REVIEW

**Comment by Riekland** [comment]
>The projectile sticks in the wound!
I think this might be the reason it's just a taser shot
Things embedded in people are absolute cancer to get out my m8

**Comment by Exxion** [comment]
Stunshells stun for WAY longer than beanbags
Also, they're physical projectiles, so ablatives and eswords don't work on them
THAT's the advantage they have over tasers

**Comment by Blithering** [comment]
I thought the point of stun shells was beanbag rounds without the damage component, so no silicons get distressed when you fire blindly into crowds. This looks to be more of a rock salt round.

**Comment by jakmak** [comment]
Exxion you can tazer someone wearing ablative by shooting their legs. And blithering the 5 damage from the stun shell was there BEFORE I made this. These things are never used because beanbag shells do the same thing and are also more plentiful while these cost a ass ton to make via protholathe and only a few maps have them in the armory round start.

**Comment by Sprok0** [comment]
@Exxion Energy swords don't have any special blocking effects against energy projectiles, they block physical ones just as well.

**Comment by SangriaMonkey** [comment]
As good as you're making them sound in your posts I rarely see anyone use them in-game.

**Comment by despotate** [comment]
> As it stands now stunshells are just tazer rounds for shotguns. 

Wrong. They will travel through glass and airlocks, cause minor brute damage, and look much more intimidating to the average greyshit. They're my favorite code green weapon for this reason and I always ask my warden for a shotty and box of stun shells roundstart.

**Comment by jakmak** [comment]
Despotate I literally have seen stun shells used once. These things function as slightly better razers which are harder to get ammo for and don't fit in backpacks/belts. As it stands now all these things are are just slightly fancier beanbag shells. Another reason they are never seen is when you get in a situation where you are using a shotgun over a tazer you won't be bothering with shitty stun ammo because if you got a shotgun outside of bartender double barrel 99% of the time your going to want/need killing power over a stun slightly better than that of a tazer

**Comment by despotate** [comment]
Stun shells are never used because no one has the balls to ask their warden for a shotgun, box of stun shells, bandolier, and box of smokes roundstart. If they were available in lockers you'd see them used more commonly. Shotguns not fitting in backpacks is actually a good thing, it means that greyshits can't conceal them if they steal one from you.

**Comment by jakmak5** [comment]
the reason why no body has the balls to round start get a shotgun and shotgun accessories is becuase that is extremley powergamey to do. What do you do when you see a sec officer walking around shotgun out wearing ablative/riot/kevlar armor 5 minutes into a round? You assume them to be a powerfag. Usually when sec is using shotguns/eguns it means shit has hit the fan and they need those lethals to survive and no one is going to bother with dinky stunshells that function nearly identically to a tazer that is more of a bitch to resupply yourself with.

**Comment by N3X15** [comment]
STOP FUCKING ARGUING OH MY GOD PRS ARE TURNING INTO A SEA OF YELLOW

**Comment by despotate** [comment]
> the reason why no body has the balls to round start get a shotgun and shotgun accessories is becuase that is extremley powergamey to do.

It's just a shotgun with stun shells, it has its drawbacks in being ASIMOV bait, a more powerful weapon for enemies if they disarm you, and more cumbersome to the uninitiated. Weapons can have their pros and cons and there's nothing wrong with asking for gear from the armory roundstart.

> What do you do when you see a sec officer walking around shotgun out wearing ablative/riot/kevlar armor 5 minutes into a round?

If someone was actually holding their shotgun and wearing that shit armor I'd assume them to be a colossal retard and a tryhard and my immediate response would to slip, disarm, stun, cuff, and disposals them to teach 'em that this level of faggotry will not be tolerated, and hand their gear in to the Captain / HoP along with an explanation of why it deserved confiscating. By the way, let's be clear just in case anyone reading this doesn't know, shotguns DO fit into the armor suit storage slot so you don't need to carry it around in your hands all the time.

**Comment by Probe1** [comment]
Why I'm against this PR:

AoE stun is incredibly powerful and shotguns have an established purpose and use.  Proliferation of shotgun use among security in non emergency situations isn't a beneficial change to the server.  It will cause more tension, more confusion, and it doesn't improve the game as a whole.  

@despotate spessman code is one thing but if you attack a security officer and go to the Captain, unless that Captains pretty bad, you're gonna get a long cell timer.

**Comment by despotate** [comment]
Long cell timer or no long cell timer it'll have been worth it. Seriously, why the fuck would anyone wear that armor?

**Comment by Blithering** [comment]
Since noone else has, and it was mentioned, could you remove that 5 brute? it would make stun shells distinct from beanbags then, being the asimov friendly shotgun ammo.

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -155,6 +155,36 @@
...
+/obj/item/projectile/bullet/stunshot/New(atom/T, var/C = 0)
+	..(T)
+	is_child = C
+
+/obj/item/projectile/bullet/stunshot/OnFired()
+	if(!is_child)
+		var/list/turf/possible_turfs = list()
+		for(var/turf/T in orange(original,1))
+			possible_turfs += T
+		for(var/I = 1; I <=4; I++)
```
Comment (PJB3005) [inline code/modules/projectiles/projectile/bullets.dm:170]: `var/I = 1 to 4`

File: code/modules/projectiles/projectile/bullets.dm [hunk 2]
```diff
@@ -155,6 +155,36 @@
...
 	weaken = 10
 	stutter = 10
+	rotate = 0
+	var/is_child = 0
+
+/obj/item/projectile/bullet/stunshot/New(atom/T, var/C = 0)
+	..(T)
+	is_child = C
+
+/obj/item/projectile/bullet/stunshot/OnFired()
```
Comment (PJB3005) [inline code/modules/projectiles/projectile/bullets.dm:165]: ...Is this also just copy pasted from buck shot?

File: code/modules/research/designs/weapons.dm [hunk 3]
```diff
@@ -257,3 +257,13 @@
 	materials = list(MAT_IRON = 12000)
 	category = "Weapons"
 	build_path = /obj/item/weapon/storage/pneumatic
+
+/datum/design/stunshellbox
+	name = "Stun Shell Box"
```
Comment (PJB3005) [inline code/modules/research/designs/weapons.dm:262]: Remove capitalization.

**Comment by ComicIronic** [comment]
Unfixed for 3 months, closing.

**Pull request closed without merging**